doc/method_documentation.rdoc

```diff
@@ -181,3 +181,38 @@ of the method.
 Methods are documented using RDoc syntax.  See the
 {RDoc Markup Reference}[https://docs.ruby-lang.org/en/master/RDoc/Markup.html#class-RDoc::Markup-label-RDoc+Markup+Reference]
 for more information on formatting with RDoc syntax.
+
+=== Output from irb
+
+Consider whether <tt># => ...</tt> in successive codeblock lines should be aligned.
+Alignment may sometimes aid readability.
+
+=== Lists
+
+A list should be preceded by and followed by a blank line.
+This is unnecessary for the HTML output, but helps in the +ri+ output.
+
+=== Auto-Links
```

Comment:
I'm not sure whether we want to have an Auto-Links section.  I think it should be left to the person writing the documentation whether or not they want auto-linking suppressed.

Reply:
I agree about "left to the person," but @stomar seemed to have a strong opposing view.  I thought it might help if we explicate the point here and reach consensus.  Tallyrand reminds us:  "If it goes without saying, it goes all the better for saying it."

Reply:
I don't think whether we auto-link is important.  If the issue is purely visual, @stomar is correct that it could be fixed with styling.  It's marginally more of a pain to explicitly avoid auto-linking when writing or editing documentation, so if we have to pick one way or the other, let's just settle it and state that we should not attempt to avoid auto-linking.

Reply:
Again, ok with me.  And again, I'm told after a year's work that what I've been doing is now considered wrong.

Reply:
Section removed.

Reply:
Removing the section goes against the discussion we just had.  So if you are OK with the consensus to not attempt to avoid auto-linking (as you indicated), you should add the section back and explicitly document that you should not attempt to avoid auto-linking.  You should also remove the change from `English` to `\English`.

Now, as I said previously, I don't think it matters much either way, so if you want to commit this with the section removed, I'm not going to object.  But it seems wrong to push for consensus, and then when you don't agree with the consensus, take actions as if the consensus was not reached.

Reply:
Didn't mean to remove b/c disagreed, meant to remove b/c you'd earlier suggested it.  I'll work this out, and thanks.

Should it be ok to suppress when the word (e.g., English, Class) doe not refer to the class?

Reply:
> Should it be ok to suppress when the word (e.g., English, Class) doe not refer to the class?

I hope so 👍

Reply:
I agree with @marcandre, if the link doesn't refer to the class, it's OK to suppress auto linking (though not required).

Reply:
Section now included.